### PR TITLE
Werewolf

### PR DESCRIPTION
**Pull request by ktrant84**
Determine if new hires are actually vampires instead of werewolves @mechamanny

### CODE REVIEW

File: ruby/vampires.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#Loop for multiple employees
+puts "How many employees will be processed today?"
+employee_process = gets.chomp.to_i
+# define loop properties
+until employee_process <= 0 do
```
Comment (mechamanny) [inline ruby/vampires.rb]: `== 0` would work here and it would be more obvious what the conditional is testing for.

File: ruby/vampires.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	employee_process -= 1	
+	#Set age_match variable for determining if age & birthday match
+	age_match = 2016 - year_born - how_old
+		#Determine vampire status
+		puts case
+		#If age matches and answer yes to one question but name isnt specific variable
+		when (age_match == 0 && (full_name != "Drake Cula" && full_name != "Tu Fang")) && (garlic_bread == "y" || health_insurance == "y")
+			print "Probably not a vampire"
+		#If age doesnt match and answer no to one question
+		when (age_match != 0) && (garlic_bread == "y" ||  health_insurance == "y")
```
Comment (mechamanny) [inline ruby/vampires.rb]: It is not incorrect to put parenthesis on `age_match != 0` but it is unecessary.

File: ruby/vampires.rb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+			print "Almost certainly a vampire"
+		#If all match but name is Drake Cula or Tu Fang
+		when (full_name == "Drake Cula") || (full_name == "Tu Fang")
+			print "Definitely a vampire"
+		else
+			print "Results Inconclusive"
+		end
+end
+#Print to let everyone be friends
+p "Actually, never mind! What do these questions have to do with anything? Let's all be friends."
```
Comment (mechamanny) [inline ruby/vampires.rb]: p returns and prints to the console, if you only want to let the user know, use `puts`. `p` is more useful for debugging.

**Comment by mechamanny** [comment]
Left some comments for you to review @ktrant84 =)

File: ruby/vampires.rb [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+			puts "Probably a vampire"
+			break
+		end
+		##End loop when allergies is done
+		until allergies == "done"
+			puts "Any other allergies? If no, type done."
+			allergies = gets.chomp	
+		end
+	#Subtract 1 employee for each completed loop
+	employee_process -= 1
```
Comment (ktrant84) [inline ruby/vampires.rb]: When I wrote it that seemed like the better place to put it then the end, now after looking at more code I would have put it after my case statement.

**Comment by ktrant84** [comment]
Thank you so much, it  was extremely helpful to have someone elses eyes to look at my code. I appreciate it.

**Comment by mechamanny** [comment]
=D

**Pull request merged**